### ISSUE TITLE
Document shell completion

### ISSUE THREAD

**Issue by michidk** [Shemnei/punktf#126]
We seem to support some kind of shell completion generation since we use `clap_complete`. The readme.md should document how to use this feature.

**Comment by Shemnei** [comment]
Good point, the documentation needs some major work. I will try to document it soonish but won't promise anything.
For now, if you are interesseted, use the `punktf completions --help` function for more information.
